features/wishlist.feature

```diff
@@ -10,6 +10,11 @@ Background:
         | 256         | today          |
         | 512         | tomorrow       |
 
+Scenario: The server is running
+    When I visit the "Home Page"
```

Comment:
Can you also add a `Given` clause?

Reply:
No need to add anything, the `given` clause is the `Given the following wishlists`.